doc/nrf/libraries/dfu/dfu_multi_image.rst

```diff
@@ -0,0 +1,33 @@
+.. _lib_dfu_multi_image:
+
+DFU Multi Image
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The DFU Multi Image library provides an API for writing a DFU Multi Image package that is being downloaded in chunks of an arbitrary size.
+The DFU Multi Image package is a simple archive file consisting of a CBOR-based header describing contents of the package, followed by a number of updates images, such as firmware images for different MCU cores.
+Images included in a DFU Multi Image package are identified by user-assigned numeric identifiers.
+The library provides a means for a user to register custom functions for writing package data that belongs to an image with a given identifier.
+That makes the library a general-purpose solution as it makes no assumptions with respect to formats of images included in a written package.
```

Comment:
```suggestion
Given that the library makes no assumptions with respect to formats of images included in a written package, it makes it a general-purpose solution for multi-image builds, for example for the :ref:`nRF5340 DK <ug_nrf5340_multi_image>`.
```

Reply:
I rephrased a bit to convey better what I meant. I would like to not make the impression that the library is only for multi-image builds (meaning multi-image firmwares).